srcpkgs/mutter/template

```diff
@@ -1,23 +1,24 @@
 # Template file for 'mutter'
 pkgname=mutter
-version=3.30.2
-revision=2
-build_style=gnu-configure
+version=3.32.0
+revision=1
 build_helper="gir"
-configure_args="--disable-schemas-compile --disable-static --enable-egl-device
- --enable-remote-desktop"
+build_style=meson
+configure_args="-Degl_device=true -Dudev=false -Dnative_backend=false
```

Comment:
Please switch udev and native_backend to true

Reply:
Please also enable egl_device for Nvidia users

Reply:
Isn't egl device is already enabled? But I did enable the other two as requested.